### DISCUSSION TITLE
Will this work on other runtimes, like React Native Hermes?

### DISCUSSION THREAD

**Discussion by andrioid** [streamich/json-joy#539] [General]
'm quite interested in doing some local-first apps, in React Native and have had problems with YJS on that front. json-joy looks to solve many of my YJS pain points, but I can't find any mentioning React Native.

Has anyone here tried using json-joy CRDT on React Native? If so, can you share some examples?

If not, then I'd appreciate hints as to what I might have to polyfill to get it working.

**Comment by streamich** [comment]
I have tried using it in React Native. Also curious to read anyones experience/feedback.
